sia-gpu-miner.c

```diff
@@ -215,15 +278,19 @@ int main(int argc, char *argv[]) {
 	cl_context context = NULL;
 	cl_program program = NULL;
 	cl_uint platformid = 0, deviceidx = 0;
-	int i, c;
+	int i;
 	unsigned cycles_per_iter;
-	char *port_number;
+	char port_number[6] = "9980";
```

Comment:
Later I read in a port and want to make sure I have 6 bytes (5 would be fine I guess) allocated so that if I copy in a higher port number, I don't write over anything.